### PR TITLE
[AddressingBundle] Added sorting countries by name in form type

### PR DESCRIPTION
**Pull request by GSadee**
| Q               | A
| --------------- | ---
| Bug fix?        | yes
| New feature?    | no
| BC breaks?      | no
| Deprecations?   | no
| Related tickets | 
| License         | MIT
| Doc PR          |

### CODE REVIEW

File: src/Sylius/Bundle/AddressingBundle/Form/Type/CountryCodeChoiceType.php [hunk 1]
```diff
@@ -64,6 +64,6 @@ private function getCountryCodes(array $countries)
             $countryCodes[$country->getCode()] = Intl::getRegionBundle()->getCountryName($country->getCode());
         }
 
-        return $countryCodes;
+        return asort($countryCodes);
```
Comment (peteward) [inline src/Sylius/Bundle/AddressingBundle/Form/Type/CountryCodeChoiceType.php]: Maybe better to sort when you retrieve from doctrine on lines 33 and 35?

Comment (GSadee) [inline src/Sylius/Bundle/AddressingBundle/Form/Type/CountryCodeChoiceType.php]: Unfortunately, I receive from doctrine only country codes, but I want to sort by country names. However I'm not sure that it is the best solution.

Comment (peteward) [inline src/Sylius/Bundle/AddressingBundle/Form/Type/CountryCodeChoiceType.php]: ah ok good point :+1:

**Comment by michalmarcinkowski** [comment]
Thanks Grzegorz! :+1:

**Pull request merged**